operator/pkg/deinit.go

```diff
@@ -84,13 +84,13 @@ func (data *deInitData) RemoteClient() clientset.Interface {
 
 // NewJobDeInitOptions calls all of DeInitOpt func to initialize a DeInitOptions.
 // if there is not DeInitOpt functions, it will return a default DeInitOptions.
-func NewJobDeInitOptions(opts ...DeInitOpt) *DeInitOptions {
-	options := defaultJobDeInitOptions()
+func NewJobDeInitOptions(options ...DeInitOpt) *DeInitOptions {
```

Comment:
DeInitOpt is different with DeInitOptions, I think `opts` is more appropriate here. 
What do you think?

Reply:
Yes,  its ok